_shows/20_21/speaking_solo.md

```diff
@@ -0,0 +1,43 @@
+---
+# Check out the Show Docs for more information
+# -- https://history.newtheatre.org.uk/docs/show/
+# Remove or comment any lines that are not being used
+
+title: NNT Presents 'Speaking Solo'
```

Comment:
Should the title be just "Speaking Solo"?

Reply:
I can change it to that